jina/serve/runtimes/helper.py

```diff
@@ -102,6 +102,8 @@ def _parse_specific_params(parameters: Dict, executor_name: str):
     def _create_aux_model_doc_list_to_list(model):
         fields: Dict[str, Any] = {}
         for field_name, field in model.__annotations__.items():
+            if field_name not in model.__fields__:
+                continue
```

Comment:
but here what will happen is that the model is not shown in the gateway.

I need an example that proves that if I deploy an Executor in a Flow with such a document class, that that info also shows up in the OpenAPI description.

Reply:
Or isn't it ur expected behavior?

Reply:
@JoanFM the way we are currently using ClassVars, they should not show up in the OpenAPI documentation. They are attributes specific to a type of input that is used internally within the service. One of the recent changes broke this pattern for us, so this change is to fix that.

Reply:
@JoanFM I added another [example/test here](https://github.com/jina-ai/jina/pull/6035/files#diff-4a9008012b5ba14919be3fd22f1e0da267ca041c67411dd0343fabb0065c203bR13-R15). Here the classvar is an attirbuted associated with the input type, but not exposed to the user.